### DISCUSSION TITLE
XBox Support

### DISCUSSION THREAD

**Discussion by nealepetrillo** [jellyfin/jellyfin#9439] [General] [answered]
Hello, 

Are there any plans for official XBox support? I understand the old Jellyfin client has been abandoned and Videotape is no longer on the MS store leaving the Edge browser and the Kodi client. Frankly, I'd like to switch from Plex but the XBox support has been fairly good - I don't think Kodi can compete and Edge tends to be fairly clunky in my experience. What are everyone's thoughts?

**Comment by nielsvanvelzen** [comment, marked as answer]
We don't have anyone right now working on an Xbox client so it's unlikely at the moment.
